### PR TITLE
Update client core

### PR DESCRIPTION
**Pull request by thecalcc**

### CODE REVIEW

**Review by dzonidoo — APPROVED** [review]

File: client/package-lock.json [hunk 1]
```diff
@@ -10498,13 +10556,18 @@
...
+                    "resolved": "https://registry.npmjs.org/whatwg-fetch/-/whatwg-fetch-2.0.4.tgz",
+                    "integrity": "sha512-dcQ1GWpOD/eEQ97k66aiEVpNnapVj90/+R+SXTPYGHpYBBypfKJEQjLrvMZ7YXbKm21gXd4NcuxUTjiv1YtLng=="
                 }
             }
         },
         "superdesk-ui-framework": {
-            "version": "3.0.80",
-            "resolved": "https://registry.npmjs.org/superdesk-ui-framework/-/superdesk-ui-framework-3.0.80.tgz",
-            "integrity": "sha512-ppOOz52Z6RXG+eZ4dPFBBj59n4/FMWzx85qRKTZTGBgwH399QkqLkMONdp81CG6Nlc5oHvsiWkFchWZvVTGBSA==",
+            "version": "3.0.82",
```
Comment (petrjasek) [inline client/package-lock.json:10568]: can we bump the minor version at least or are you sure that all those changes are ok also for sd 2.6?

Comment (petrjasek) [reply client/package-lock.json:10568]: well now it's late anyhow but would be good to bump the version in ui-framework repo

Comment (thecalcc) [reply client/package-lock.json:10568]: I've changed the `Menu` component, adjusting the max-hegith and how the menu behaves when it gets too large, so I'm pretty confident nothing will break, it's going to actually improve

Comment (thecalcc) [reply client/package-lock.json:10568]: Ok next version will be 3.1..something

**Pull request merged**